ecosystem/sep-0038.md

```diff
@@ -7,9 +7,9 @@ Author: Jake Urban <@jakeurban> and Leigh McCulloch <@leighmcculloch>
 Track: Standard
 Status: Draft
 Created: 2021-04-09
-Updated: 2023-08-24
+Updated: 2024-08-15
 Discussion: https://github.com/stellar/stellar-protocol/issues/901
-Version 2.2.0
+Version 2.2.1
```

Comment:
I think this should be a minor version update since its additive, right?
```suggestion
Version 2.3.0
```

Reply:
Yes.